src/app/codegen-data-model/CodegenDataModel_Read.cpp

```diff
@@ -138,13 +88,30 @@ struct ShortPascalString
 {
     using LengthType                        = uint8_t;
     static constexpr LengthType kNullLength = 0xFF;
+
+    static size_t GetLength(ByteSpan buffer)
+    {
+        VerifyOrDie(buffer.size() >= 1);
+        // NOTE: we do NOT use emberAfLongStringLength because that will result in 0 length
+        //       for null sizes (i.e. 0xFF is translated to 0 and we do not want that here)
```

Comment:
We don't use emberAfLongStringLength because it ... would just be the wrong thing to use for a _short_ string as here.

Please fix this comment.